vortex-datafusion/src/lib.rs

```diff
@@ -10,6 +10,8 @@ use vortex::stats::Precision;
 mod convert;
 pub mod persistent;
 
+pub use persistent::*;
```

Comment:
Now that's the only module in the repo, figured I'll re export everything there.